src/main/java/graphql/schema/validation/Validator.java

```diff
@@ -1,31 +1,40 @@
 package graphql.schema.validation;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
 import graphql.schema.GraphQLFieldDefinition;
 import graphql.schema.GraphQLFieldsContainer;
 import graphql.schema.GraphQLOutputType;
 import graphql.schema.GraphQLSchema;
+import graphql.schema.GraphQLType;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
 public class Validator {
 
     private final Set<GraphQLOutputType> processed = new HashSet<>();
-    
+
     public Set<ValidationError> validateSchema(GraphQLSchema schema) {
         ValidationErrorCollector validationErrorCollector = new ValidationErrorCollector();
         List<ValidationRule> rules = new ArrayList<>();
         rules.add(new NoUnbrokenInputCycles());
-        
+        rules.add(new ObjectsImplementInterfaces());
+
+        List<GraphQLType> types = schema.getAllTypesAsList();
+        types.forEach(type -> {
+            for (ValidationRule rule : rules) {
+                rule.check(type, validationErrorCollector);
+            }
+        });
+
         traverse(schema.getQueryType(), rules, validationErrorCollector);
         if (schema.isSupportingMutations()) {
```

Comment:
yes it is missing that.  We can do that in another PR.  

https://github.com/graphql-java/graphql-java/issues/414